### ISSUE TITLE
Apply one-click-studio-invites  in popup

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6226]
### Why this would be helpful

Sometimes you want to accept studio invitations easily from the messages page, so the one-click-studio-invites addon was created. However, if you use the popup, then the addon does nothing.

### My suggestion

Apply one-click-studio-invites to the popup as well.

### Possible alternatives

We could add an option to scratch-messaging too, but I think it would make more sense to have it in one-click-studio-invites.

### Additional context

It should probably look better, but something like this:
![An accept button below a studio invite.](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/a0fe6b20-4ce3-4ad8-a92d-14cad5528083)

**Comment by WorldLanguages** [comment]
Related: Ctrl+Enter to post already works in the messaging popup.

**Comment by DNin01** [comment]
I think this could be built into the popup and not controlled by an addon.

**Comment by WorldLanguages** [comment]
> I think this could be built into the popup and not controlled by an addon.

Yeah, we could just add it directly to scratch-messaging with no option to turn it off, as long as we prompt the user to confirm before actually joining the studio.

**Comment by penguinmoose** [comment]
> > I think this could be built into the popup and not controlled by an addon.
> 
> Yeah, we could just add it directly to scratch-messaging with no option to turn it off, as long as we prompt the user to confirm before actually joining the studio.

However, the popup should be somewhat discrete and shouldn't take up too much space.
